automates/text_reading/src/main/scala/org/clulab/aske/automates/mentions/CrossSentenceEventMention.scala

```diff
@@ -33,6 +33,20 @@ class CrossSentenceEventMention(
             foundBy: String
           ) = this(Seq(label), mkTokenInterval(trigger, arguments), trigger, arguments, paths, sentence, sentences, document, keep, foundBy, Set.empty)
 
+  def copy(
```

Comment:
@alicekwak, the changes look good! A quick question: what is this method for?

Reply:
Never mind. Looks like a method that you can use to copy a mention with some modifications.

Reply:
That's correct!

Reply:
@alicekwak : Can you please add an explicit note to capture the descriptive summary @maxaalexeeva gave?